scss/_type.scss

```diff
@@ -26,10 +26,10 @@ h6, .h6 { @include font-size($h6-font-size); }
 }
 
 // Type display classes
-@each $index, $values in $displays {
+@each $index, $display-type in $display-types {
   .display-#{$index} {
-    @include font-size(#{nth($values, 1)});
-    font-weight: #{nth($values, 2)};
+    @include font-size(map-get($display-type, font-size));
+    font-weight: map-get($display-type, font-weight);
```

Comment:
I'm getting an "Expected numeric font-weight notation" css-lint error in the map-get function, but all seems right and the compiling output is also right... so I don't know...

Reply:
You can disable this with `// stylelint-disable-next-line font-weight-notation`. Stylelint doesn't know what the `map-get` result will be, so you need to explicitly disable the rule on that line.